homeassistant/components/victron_ble/strings.json

```diff
@@ -0,0 +1,276 @@
+          "over_charge": "Overcharge protection",
+          "input_voltage": "Input overvoltage",
+          "input_current": "Input overcurrent",
+          "input_power": "Input overpower",
+          "input_shutdown_voltage": "Input shutdown (battery overvoltage)",
+          "input_shutdown_current": "Input shutdown (current flow during off mode)",
+          "input_shutdown_failure": "PV input failed to shutdown",
+          "inverter_shutdown_41": "Inverter shutdown (PV isolation)",
+          "inverter_shutdown_42": "[%key:component::victron_ble::entity::sensor::charger_error::state::inverter_shutdown_41%]",
+          "inverter_shutdown_43": "Inverter shutdown (Ground fault)",
+          "inverter_overload": "Inverter overload",
+          "inverter_temperature": "Inverter temperature too high",
+          "inverter_peak_current": "Inverter peak current",
+          "inverter_output_voltage_a": "Inverter output voltage",
+          "inverter_output_voltage_b": "[%key:component::victron_ble::entity::sensor::charger_error::state::inverter_output_voltage_a%]",
+          "inverter_self_test_a": "Inverter self-test failed",
+          "inverter_self_test_b": "[%key:component::victron_ble::entity::sensor::charger_error::state::inverter_self_test_a%]",
+          "inverter_self_test_c": "[%key:component::victron_ble::entity::sensor::charger_error::state::inverter_self_test_a%]",
+          "inverter_ac": "Inverter AC voltage on output",
+          "communication": "Communication warning",
+          "synchronisation": "Synchronized charging device configuration issue",
+          "bms": "BMS connection lost",
+          "network_a": "Network misconfigured",
+          "network_b": "[%key:component::victron_ble::entity::sensor::charger_error::state::network_a%]",
+          "network_c": "[%key:component::victron_ble::entity::sensor::charger_error::state::network_a%]",
+          "network_d": "[%key:component::victron_ble::entity::sensor::charger_error::state::network_a%]",
+          "pv_input_shutdown_80": "PV input shutdown",
+          "pv_input_shutdown_81": "[%key:component::victron_ble::entity::sensor::charger_error::state::pv_input_shutdown_80%]",
+          "pv_input_shutdown_82": "[%key:component::victron_ble::entity::sensor::charger_error::state::pv_input_shutdown_80%]",
+          "pv_input_shutdown_83": "[%key:component::victron_ble::entity::sensor::charger_error::state::pv_input_shutdown_80%]",
+          "pv_input_shutdown_84": "[%key:component::victron_ble::entity::sensor::charger_error::state::pv_input_shutdown_80%]",
+          "pv_input_shutdown_85": "[%key:component::victron_ble::entity::sensor::charger_error::state::pv_input_shutdown_80%]",
+          "pv_input_shutdown_86": "[%key:component::victron_ble::entity::sensor::charger_error::state::pv_input_shutdown_80%]",
+          "pv_input_shutdown_87": "[%key:component::victron_ble::entity::sensor::charger_error::state::pv_input_shutdown_80%]",
```

Comment:
I think one thing to keep in mind is that this will give the user the option between `PV input shutdown`, `PV input shutdown`, `PV input shutdown`, `PV input shutdown`, `PV input shutdown`, and `PV input shutdown`. So if there is no difference between these values, I would recommend to map them to a single value as otherwise users need to differentiate on the 8 different states in automations